worker/workercmd/guardian.go

```diff
@@ -102,13 +107,14 @@ func (cmd *WorkerCommand) guardianRunner(logger lager.Logger) (ifrit.Runner, err
 
 	gdnArgs := append(gdnConfigFlag, append([]string{"server"}, gdnServerFlags...)...)
 
-
-	flagsInConfig, err := getGdnFlagsFromConfig(cmd.Guardian.BinaryFlags.Config.Path())
-	if err != nil {
-		return nil, err
+	if cmd.Guardian.Config != "" {
+		configFlags, err := getGdnFlagsFromConfig(cmd.Guardian.Config.Path())
```

Comment:
https://ci.concourse-ci.org/builds/2881001#L5f6df0f7:67

```suggestion
	var configFlags GdnBinaryFlags
	if cmd.Guardian.Config != "" {
		configFlags, err = getGdnFlagsFromConfig(cmd.Guardian.Config.Path())
```

Reply:
😅 thats why u should always build before pushing.. haha